tests/acceptance/fixtures/responses/universal.js

```diff
@@ -484,3 +523,15 @@ export const MockedUniversalSearchRequest = RequestMock()
     return urlRegex.test(request.url) && request.method === 'get';
   })
   .respond(UniversalSearchResponse, 200, CORSHeaders);
+
+export const MockedUniversalAutoCompleteRequest = RequestMock()
+  .onRequestTo(async request => {
+    const urlRegex = /^https:\/\/liveapi-cached.yext.com\/v2\/accounts\/me\/answers\/autocomplete/;
```

Comment:
I was hoping we could do
```ts
.onRequestTo({ url: /regex/, method: 'GET', isAjax: true })
```
though I'm not sure if this would work?

Reply:
I don't think that will work unfortunately. The first arg is a string, not a regex. I believe the string is a literal match as well.